compiler/Cargo.toml

```diff
@@ -1,21 +1,21 @@
 [package]
 name = "ttrpc-compiler"
-version = "0.7.0"
-edition = "2018"
-authors = ["The AntFin Kata Team <kata@list.alibaba-inc.com>"]
-license = "Apache-2.0"
+version = "0.8.0"
+edition.workspace = true
+license.workspace = true
+authors.workspace = true
+readme = "README.md"
 keywords = ["compiler", "ttrpc", "protobuf"]
 description = "ttRPC compiler for ttrpc"
 categories = ["network-programming"]
 repository = "https://github.com/containerd/ttrpc-rust/tree/master/compiler"
 homepage = "https://github.com/containerd/ttrpc-rust/tree/master/compiler"
-readme = "README.md"
 
 [dependencies]
 # lock home to avoid conflict with latest version
 home = "=0.5.9"
-protobuf = "2.27.1"
-protobuf-codegen = "2.27.1"
+protobuf.workspace = true
```

Comment:
What are the benefits of writing like this?

Reply:
Having protobuf as a workspace dependency ensures that all crates in the workspace use the same version of protobuf.
This could also be expressed as
```toml
protobuf = { workspace = true }
```